types/firestore.d.ts

```diff
@@ -1964,13 +1962,16 @@ declare namespace FirebaseFirestore {
    * Firestore v1beta1 RPCs.
    * @deprecated Use v1 instead.
    */
-  export const v1beta1: any;
+  export const v1beta1: import('./v1beta1/firestore_client').FirestoreClient;
```

Comment:
I am a bit torn about using the default export here, which is really an anti-pattern and not used in the export below. I think no matter how we export this, we have a chance of breaking users, but the clean way would be to export this as { FirestoreClient: import(...).FirestoreClient }. That both works is technical debt that I would like to not carry forward.

Reply:
done.